reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -425,7 +426,10 @@ class StaticHtmlReporter : Reporter() {
     private fun TBODY.evaluatorRow(error: ResolvableIssue) {
         val cssClass = when {
             error.isResolved -> "ort-resolved"
-            else -> "ort-error"
+            error.severity == Severity.ERROR -> "ort-error"
```

Comment:
This is different from my proposal in an important regard: By not doing the `when` on `error.severity` the Kotlin compiler cannot verify that all enum values are handled anymore.

Reply:
Adjusted according to you proposal.